myosuite/utils/__init__.py

```diff
@@ -0,0 +1,16 @@
+import importlib.util
+
+if importlib.util.find_spec("gymnasium"):
+    import gymnasium as gg
+elif importlib.util.find_spec("gym"):
+    import gym as gg
+else:
+    raise Exception("Gym/Gymnasium not found! You can install it with `pip install gymnasium`")
+
+class gym(): pass
+
+# https://stackoverflow.com/questions/21434332/how-to-extend-inheritance-a-module-in-python
+for i in gg.__all__ + ['__version__', 'envs', 'utils', 'spaces']:
```

Comment:
@Vittorio-Caggiano -- I'm curious why you are explicitly adding `['envs', 'utils', 'spaces']`. Aren't they present by default? Maybe I'm missing something.

Reply:
`__all__` gets only the variables but the modules are not provided. Those modules are the ones specifically needed for myosuite.
Actually, a more comprehensive solution could be to add all modules with something like:
`[m[0] for m in inspect.getmembers(gg, predicate=inspect.ismodule)]`

Reply:
I believe I have found a much cleaner answer. Giving it a try now. Will update in sometime

Reply:
Got it working. See this commit - https://github.com/vikashplus/robohive/commit/1052b24d036337496487c0c558174a60b7050c17